internal/ent/schema/controlfields.go

```diff
@@ -25,10 +25,12 @@ var controlFields = []ent.Field{
 		Optional().
 		Unique().
 		Comment("external auditor id of the control, can be used to map to external audit partner mappings"),
-	field.String("status").
+	field.Enum("status").
+		GoType(enums.ControlStatus("")).
 		Optional().
+		Default(enums.ControlStatusPreparing.String()).
 		Annotations(
-			entgql.OrderField("status"),
+			entgql.OrderField("STATUS"),
```

Comment:
fyi, all other enums use upper case for the ordering so this is a consistent change.